scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -131,6 +98,39 @@ def update_db(version):     # pre-upgrade DB Update
     finally:
         close_connections()
 
+
+def migrate_audit_data():
```

Comment:
function name like update_audit_db will look more appropriate !

Reply:
ok sure will do it